### DISCUSSION TITLE
Save selected map layers of a theme

### DISCUSSION THREAD

**Discussion by ppete2** [devemux86/cruiser#244] [Q&A] [answered]
Regarding Cruiser for Desktop PC, but also for Android:

Please save the selected map layers of a certain Design. Right now, after we meanwhile switched to another Design, and switch back to the original Design, we have to select the desired map layers again although we already selected them in the past.

**Comment by devemux86** [comment, marked as answer]
@ppete2 thanks for the suggestion.

It is possible, just a bit complicated, as the map layers are provided by each loaded map theme.
Currently when we first load a map theme, the map layers are set to their defaults.

Cruiser map layer dialogs are generated automatically.
